aws/network-primitives/network.tf

```diff
@@ -0,0 +1,193 @@
+resource "aws_vpc" "gemini_vpc" {
```

Comment:
Should it be called `gemini_*` if it is in primitives (here and below)?

Reply:
That is the name of the resource, the naming is correct and can't be set dynamically afaik.

Reply:
I'd argue the name is not correct. This file is `aws/network-primitives/network.tf`, meaning it is used by all networks as a primitive, not just by Gemini networks, so should be named accordingly, otherwise looks misleading to me.

Reply:
I just used "network" instead, and the "name" tag will set the actual network name to identify it.